verifiers/fflonk/src/benchmarking.rs

```diff
@@ -72,7 +73,7 @@ mod benchmarks {
     #[benchmark]
     fn register_vk() {
```

Comment:
Should we consider having a max_size vk in terms of the weight required to save it in storage ? The difficulty here is that we would need to "bypass" the validate_vk

Reply:
Also one can consider getting fancier with the benches, e.g. register_vk_not_registered, register_vk_already_registered, unregister_vk_only_ref, unregister_vk_many_ref, but maybe is overdoing

Reply:
- For the first comment, please see the point above.
- For the second comment, I agree that it would be possible to benchmark all of those paths. However, this would require to do the following:
  * Since at the moment benchmarks are not "modular", we would need to write those benchmark variants for every single verifier pallet
  * Since it's not possible to distinguish the specific variant from the inputs without actually executing the extrinsic, we would need to use the `DispatchResultWithPostInfo`
Considering that, in normal workflows, errors in these extrinsic would be catched before the extrinsic gets included in a block, I don't think that the effort is really worth it